src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24864.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue24864 : _IssuesUITest
+	{
+		public override string Issue => "Button control: inconsistent border visibility behaviour iOS vs Android when BackgroundColor is Transparent";
+
+		public Issue24864(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.Button)]
+		public void ButtonShouldHaveTransparentBorder()
+		{
+			App.WaitForElement("label");
```

Comment:
This test is failing on Catalyst. Line 19, just waiting for the Label. Could you take a look? Let me know if need help with something.
<img width="1156" alt="image" src="https://github.com/user-attachments/assets/57ad46b0-ce0d-47ae-8c17-501fc61c3d35">

Reply:
The same thing happens on Windows. I wonder what is the reason since the label that test is waiting for is displayed straightaway🤔

```c#
[Test]
[Category(UITestCategories.Button)]
public void ButtonShouldHaveTransparentBorder()
{
	App.WaitForElement("label");
	VerifyScreenshot();
}
```

```c#
Content = new VerticalStackLayout()
{
	new Button
	{
		BorderWidth = 10,
		TextColor = Colors.Black,
		BackgroundColor = Colors.Transparent,
		Text = "Hello World",
		Margin = new Thickness(10)
	},
	new ImageButton
	{
		BorderWidth = 1,
		BackgroundColor = Colors.Transparent,
	},
	new Label()
	{
		TextColor = Colors.White,
		AutomationId = "label"
	}
};
```